HelperFunctions.ps1

```diff
@@ -1354,3 +1354,44 @@ function Write-GroupEnd {
         $bcContainerHelperConfig.IsAzureDevOps { Write-Host "##[endgroup]"; break }
     }
 }
+
+function CopySymbolsFromContainer {
+    Param(
+        [string] $containerName,
+        [string] $containerSymbolsFolder
+    )
+
+    Invoke-ScriptInBcContainer -containerName $containerName -scriptblock { Param($appSymbolsFolder)
+        if (Test-Path "C:\Extensions\*.app") {
+            $paths = @(
+                "C:\Program Files\Microsoft Dynamics NAV\*\AL Development Environment\System.app"
+                "C:\Extensions\*.app"
+            )
+        }
+        else {
+            $paths = @(
+                "C:\Program Files\Microsoft Dynamics NAV\*\AL Development Environment\System.app"
+                "C:\Applications.*\Microsoft_Application_*.app,C:\Applications\Application\Source\Microsoft_Application.app"
+                "C:\Applications.*\Microsoft_Base Application_*.app,C:\Applications\BaseApp\Source\Microsoft_Base Application.app"
+                "C:\Applications.*\Microsoft_System Application_*.app,C:\Applications\System Application\source\Microsoft_System Application.app"
+                "C:\Applications.*\Microsoft_Business Foundation_*.app,C:\Applications\BusinessFoundation\source\Microsoft_Business Foundation.app"
+            )
+        }
+        $paths | ForEach-Object {
+            $appFiles = $_.Split(',')
+            $appFile = ""
+            if (Test-Path -Path $appFiles[0]) {
+                $appFile = $appFiles[0]
+            }
+            elseif (Test-Path -path $appFiles[1]) {
+                $appFile = $appFiles[1]
+            }
```

Comment:
Can be simplified into something like this:

$appFile = $_ -split "," | Where-Object { Test-Path ($_) } | Select -First 1